fbw-common/src/wasm/systems/systems/src/hydraulic/command_sensor_unit.rs

```diff
@@ -0,0 +1,336 @@
+use std::time::Duration;
+
+use crate::simulation::{
+    InitContext, Read, SimulationElement, SimulatorReader, UpdateContext, VariableIdentifier,
+};
+
+// This type is used to represent the position of a 5bit switch. Therefore, the
+// value must always be <= 11111.
+type SwitchPattern = u8;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum CSU {
+    Conf0 = 0,
+    Conf1,
+    Conf2,
+    Conf3,
+    ConfFull,
+    OutOfDetent,
+    Fault,
+    Misadjust, // This is triggered when flaps lever moves from 0 to 4 or viceversa
+               // with no intermediate steps. Not implemented because it can impact the simulation.
+}
+impl CSU {
+    pub fn is_valid(value: CSU) -> bool {
+        matches!(
+            value,
+            CSU::Conf0 | CSU::Conf1 | CSU::Conf2 | CSU::Conf3 | CSU::ConfFull
+        )
+    }
```

Comment:
```suggestion
    fn is_valid(&self) -> bool {
        matches!(
            self,
            CSU::Conf0 | CSU::Conf1 | CSU::Conf2 | CSU::Conf3 | CSU::ConfFull
        )
    }
```
Then you can directly call `.is_valid()` on the type.